core/src/main/scala/quasar/destination/redshift/RedshiftFlow.scala

```diff
@@ -59,6 +59,21 @@ final class RedshiftFlow[F[_]: ConcurrentEffect: ContextShift: Timer: MonadResou
     args: Flow.Args)
     extends Flow[F] with Logging {
 
+  type Stage = BlobPath
+
+  def stage(bytes: Stream[F, Byte]): Resource[F, BlobPath] = {
```

Comment:
Maybe define this as `def stage(bytes: Stream[F, Byte]): Resource[F, Stage]` instead?
And `ingest` below as `def ingest(uploaded: Stage): F[Unit]`
Just to make the names match up with the def of `Flow`.

Reply:
:+1: I think `copyInto` should use `Stage` too then